lib/applications/schema.ts

```diff
@@ -321,6 +324,7 @@ export default `
     addressLine2: String
     city: String!
     postalCode: String!
+    receiveEmailUpdates: Boolean
```

Comment:
```suggestion
```
We should remove this boolean from `CreateReplacementApplicationInput` since the checkbox isn't present for replacement applications